coreos-base/coreos/coreos-0.0.1.ebuild

```diff
@@ -75,9 +75,7 @@ RDEPEND="${RDEPEND}
 	)
 	etcd_protocols_2? (
 		dev-db/etcd:2
-		amd64? (
-			app-admin/etcd-wrapper
-		)
+		app-admin/etcd-wrapper
```

Comment:
The `coreos` ebuild revision number should probably be bumped for this.  (The symlink in the same directory.)  Maybe the `coreos-base/coreos` changes could be in their own commit.